http-ratelimiting/src/headers.rs

```diff
@@ -513,6 +600,8 @@ fn header_str(name: HeaderName, value: &[u8]) -> Result<&str, HeaderParsingError
 
 #[cfg(test)]
 mod tests {
+    use crate::headers::RatelimitScope;
+
```

Comment:
i think at some point we should decide what our import conventions are for modules and for tests.

Reply:
To be honest I just let rust-analyzer import this